### ISSUE TITLE
Did this amazing extension finally die?

### ISSUE THREAD

**Issue by Hillgrove** [nliautaud/trakttvstats#49]

**Comment by AmineI** [comment]
Although the original repository owner didn't work on it for a while, the extension and its features are working fine, as far as I know, and you're welcome to report issues.

**Comment by Hillgrove** [comment]
Lots of stuff (all?) doesn't work.

Just taking from the 4 images on the frontpage of this github here are the things not working:

1. Personal Filmography Filtering (no fancy pillar diagrams etc)
2. Career Graphs and User statistics (no fancy pillar diagrams etc)
3. Localized titels, posters, synopsis and release dates
4. Custom External links (Layout Improvements seem to work, or Trakt has implemented those by themselves.

**Comment by AmineI** [comment]
Can you tell me which version are you using ? I know for a fact that 3. and 4. definitely work fine here, along with the improvements I made in #47 . Not sure about the 2 firsts

**Comment by Hillgrove** [comment]
I'm using the one on the Chrome Webstore (0.4 (0.4.1.1)

https://chrome.google.com/webstore/detail/trakt-improver/khhjjcbdknmdbdhgkbnldplgoppeehce

**Comment by AmineI** [comment]
You can use the development version on the chrome web store https://chrome.google.com/webstore/detail/ppeoegnflhlfjkclghpfkgpgpbodlaaa (Note that this link is official, it is right next to the stable one on the readme)

**Comment by Hillgrove** [comment]
just tried the dev version (your link)

And I still don't get the fancy diagrams. Tried on a random actors page.

**Comment by AmineI** [comment]
Yes, as I said I didn't know about these 2 firsts points, thanks for reporting, I'll give it a look. However, can you confirm that 3 & 4 are indeed working on your end ?

**Comment by Hillgrove** [comment]
after re-installing the dev version again, I can confirm 3 & 4 works.

**Comment by AmineI** [comment]
I think I fixed it, but it won't be merged for a while (if ever) and thus not sent to the chrome webstore. I'll open a PR.

**Comment by AmineI** [comment]
There. I tested it a bit but I'm not sure what to expect, you can try that version and tell me if you notice something still not working.

It could also be helpful for future reference to change this issue's title to something that actually mentions the issue :) !

**Comment by nliautaud** [comment]
Hi ! I'm back :)

I indeed left this project aside following the trakt.tv update that implemented some of the thoughts in this extension.
I felt that it would probably need a fundamental design pass to analyze the difference in functionalities and adapt this extension features to the new categorized credits, truncated biography and synced percentages in the viewed/collected buttons.

But for the meantime (if this time come), @AmineI work should be merged for sure, and the v0.5 should get out of dev state in overall. I'm in the process.
